MVPSt_2023_Jan/MVPSt_2023_Jan/Utilities/CommonDriver.cs

```diff
@@ -1,34 +1,50 @@
-﻿using OpenQA.Selenium;
+﻿using MVPSt_2023_Jan.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
 
 namespace MVPSt_2023_Jan.Utilities
 {
     public class CommonDriver
     {
-        public IWebDriver driver;
+        public static IWebDriver driver;
 
+        // Nunit Hooks
         //[SetUp]
         //[OneTimeSetUp]
```

Comment:
uncomment the hooks https://codebun.com/how-to-implement-hooks-in-specflow-c/ have look at that link for hooks, once you uncomment this you don't need to call its in your stepDef. See comment on stepDef class

Reply:
Added Specflow [BeforeScenario] to CommonDriver to perform SetupDriver commands